src/common/buffer.hpp

```diff
@@ -284,104 +284,85 @@ class RxCounters {
     }
 };
 
-class Frame_stats {
+/**
+//  * This class stores the metrics coresponding to a Frame.
+//  * Specifically, it contains a) the number of symbols per frame and b) the number of tasks per symbol, per frame.
+//  */
```

Comment:
Let's use 
```suggestion
    /**
      * @brief
      * @param
      */
```
as the format for documentation for the consistency with other files.